### ISSUE TITLE
ERROR: Unexpected exception executing [Rscript ...

### ISSUE THREAD

**Issue by ivan108** [UCSF-Costello-Lab/LG3_Pipeline#30]
While running _run_Merge_QC got this:

Exception in thread "main" net.sf.picard.PicardException: Unexpected exception executing [Rscript /scratch/jocostello/Y00600t_tmp/script7931479037635108063.R /scratch/jocostello/Y00600t/Y00600t.merged.quality_by_cycle_metrics /scratch/jocostello/Y00600t/Y00600t.merged.quality_by_cycle.pdf Y00600t.merged.sorted.bam]
   at net.sf.picard.util.ProcessExecutor.execute(ProcessExecutor.java:102)
   at net.sf.picard.util.ProcessExecutor.execute(ProcessExecutor.java:78)
   at net.sf.picard.util.RExecutor.executeFromFile(RExecutor.java:73)
   at net.sf.picard.util.RExecutor.executeFromClasspath(RExecutor.java:55)
   at net.sf.picard.analysis.MeanQualityByCycle.finish(MeanQualityByCycle.java:176)
   at net.sf.picard.analysis.SinglePassSamProgram.makeItSo(SinglePassSamProgram.java:145)
   at net.sf.picard.analysis.CollectMultipleMetrics.doWork(CollectMultipleMetrics.java:107)
   at net.sf.picard.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:177)
   at net.sf.picard.cmdline.CommandLineProgram.instanceMainWithExit(CommandLineProgram.java:119)
   at net.sf.picard.analysis.CollectMultipleMetrics.main(CollectMultipleMetrics.java:85)
Caused by: java.io.IOException: Cannot run program "Rscript": java.io.IOException: error=2, No such file or directory
   at java.lang.ProcessBuilder.start(Unknown Source)
   at java.lang.Runtime.exec(Unknown Source)
   at java.lang.Runtime.exec(Unknown Source)
   at net.sf.picard.util.ProcessExecutor.execute(ProcessExecutor.java:93)
   ... 9 more
Caused by: java.io.IOException: java.io.IOException: error=2, No such file or directory
   at java.lang.UNIXProcess.<init>(Unknown Source)
   at java.lang.ProcessImpl.start(Unknown Source)
   ... 13 more

Basically Java doesn't see Rscript, needed to generate PDF QC plot...

**Comment by ivan108** [comment]
Just checked, in the old code I was setting up path to R in Merge_QC.sh:

PATH=/opt/R/R-latest/bin/R:$PATH

**Comment by HenrikBengtsson** [comment]
Ok, my mad. I might have taken that out by mistake.  Fell free to add it back - seems necessary

**Comment by ivan108** [comment]
Actually the path was there but it was wrong, so my bad. I seems the R location was taken from .bashrc in the past. Anyway, it works now..
